protob/messages.proto

```diff
@@ -912,6 +922,102 @@ message CosiSignature {
 	optional bytes signature = 1;		// Signature
 }
 
+///////////////////
+// Iota messages //
+///////////////////
+
+/**
+ * Request: Show seed on the device
+ * @next Success
+ * @next Failure
+ */
+message IotaShowSeed {
+}
+
+/**
+ * Request: Get address counter
+ * @next IotaGetAddressCounter
+ */
+message IotaGetAddressCounter {
+}
+
+/**
+ * Returns the address counter as currently set in storage
+ * @prev IotaGetAddressCounter
+ */
+message IotaAddressCounter {
+	required uint32 address_counter = 1;
```

Comment:
Never use `required`, always use `optional`.

Reply:
Is there a specific reason for that?

Reply:
Explained here https://developers.google.com/protocol-buffers/docs/proto#specifying-field-rules

Furthermore, in Proto3, all fields are `optional`

Reply:
In that case my messages are not the only ones that don't comply. Anyway, I'll change them to optional.

Reply:
The old messages that do not comply unfortunately cannot be changed (for same reason stated in the link! :smile:)